frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewSubformLayoutSet/CreateNewSubformLayoutSet.tsx

```diff
@@ -42,13 +44,14 @@ export const CreateNewSubformLayoutSet = ({
           label={t('ux_editor.component_properties.subform.created_layout_set_name')}
           value={newSubform}
           size='sm'
-          onChange={handleChange}
+          disabled={isPendingLayoutSetMutation}
+          onChange={(e) => handleChange(e.target.value)}
           error={nameError}
         />
         <StudioButton
           className={classes.savelayoutSetButton}
-          icon={<CheckmarkIcon />}
-          onClick={handleCreateSubform}
+          icon={saveIcon}
+          onClick={() => createSubform({ layoutSetName: newSubform, onSubformCreated })}
```

Comment:
Should we refactor this to a `handleClick` function?

Reply:
Same as above 🙂